python/lsst/ip/isr/isrMock.py

```diff
@@ -532,9 +546,20 @@ def makeImage(self):
             return exposure
 
     # afw primatives to construct the image structure
-    def getCamera(self):
+    def getCamera(self, isForAssembly=False):
         """Construct a test camera object.
 
+        Parameters
+        -------
+        isForAssembly : `bool`
+            If True, construct a camera with "super raw"
+            orientation (all amplifiers have LL readout
+            corner but still contains the necessary flip
+            and offset info needed for assembly. This is
+            needed if isLsstLike is True. If False, return
+            a camera with bboxes flipped and offset to the
+            correct orientation given the readout corner.
```

Comment:
First of all, missing closing parentheses. Second, I think it should say “needed for testing camera assembly” and “this setting is only needed if isLsstLike is True, otherwise it has no effect”